packages/vats/src/core/chain-behaviors.js

```diff
@@ -289,7 +289,13 @@ harden(makeBridgeManager);
  * }} powers
  */
 export const makeChainStorage = async ({
-  consume: { bridgeManager: bridgeManagerP, loadVat },
+  consume: {
+    bridgeManager: bridgeManagerP,
+    loadVat,
+    // provisioning is here to attempt delaying execution for avoiding failures like
+    // https://github.com/Agoric/agoric-sdk/runs/6728088019?check_suite_focus=true
```

Comment:
copy 1 or 2 of the most relevant lines, please? ci log URLs have a limited lifetime.

```suggestion
    // Error#1: historical inaccuracy in replay of v2 [vatAdmin]
    // https://github.com/Agoric/agoric-sdk/runs/6728088019?check_suite_focus=true
```